src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -1060,9 +1060,9 @@ def iot_central_app_create(
               sku=appSku,
               template=template)
 
-    createResult = client.apps.create_or_update(
-        resource_group_name, app_name, app)
-    return createResult
+    if no_wait:
+        return client.apps.create_or_update(resource_group_name, app_name, app)
```

Comment:
Typically we should use sdk_no_wait() to wrap the operation as shown in 
https://github.com/Azure/azure-cli/blob/8728d8133169389a5c9ac09791c9a09dd58fb224/src/azure-cli/azure/cli/command_modules/vm/custom.py#L923-L924

Reply:
ok